c7n/offhours.py

```diff
@@ -212,10 +212,8 @@ def __call__(self, i):
     def process_current_time(self, i, parts):
         try:
             parsed = ScheduleParser().parse(parts[0])
-            if parsed is None:
-                raise ValueError("malformed downtime tag")
         except Exception as e:
-            log.warning("downtime tag is malformed or empty, using defaults")
+            log.warning("Parsing Error: %s. Using default values" % e)
```

Comment:
Will display the thrown exception from `ScheduleParser().parse()` and continue with default behavior.